client/state/selectors/test/get-embed.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import getEmbed from 'state/selectors/get-embed';
+
+describe( 'getEmbed()', () => {
+	const siteId = 12345678;
+	const url = 'https://www.facebook.com/20531316728/posts/10154009990506729/';
+	const embed = {
+		body: 'something',
+		scripts: { foo: 'bar' },
+		styles: {},
+	};
+	const state = {
+		embeds: {
+			urlItems: {
+				[ siteId ]: {
+					[ url ]: embed,
+				},
+			},
+		},
+	};
+
+	test( 'should return null if the site is unknown', () => {
+		expect( getEmbed( state, 123, url ) ).toBeNull();
+	} );
+
+	test( 'should return null if the URL is unknown', () => {
+		expect( getEmbed( state, 123, 'something' ) ).toBeNull();
```

Comment:
Do we want to use the correct `siteId` here and so make sure that `null` is returned in response to an unknown URL?

Reply:
Good catch! Fixed in https://github.com/Automattic/wp-calypso/pull/38608/commits/98f18801d8586efad4e643eaf0af71e054106374